<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1072</width>
    <height>748</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Niched Pareto Genetic Algorithm (NPGA) - Tomasz Bartos, Rados≈Çaw Zwolski</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>160</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>110</y>
      <width>331</width>
      <height>91</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>541</width>
      <height>66</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLineEdit" name="function2"/>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="function1"/>
     </item>
    </layout>
   </widget>
   <widget class="QCustomPlot" name="figurePlot" native="true">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>220</y>
      <width>591</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="resultsTable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>411</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>650</y>
      <width>1031</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>110</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>tdom</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tdom">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>110</y>
      <width>59</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>6</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Population size</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="maxGenertation">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>40</y>
      <width>59</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>6</y>
      <width>132</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Crossing probability</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="crossingProbability">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>6</y>
      <width>59</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="sigma">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>74</y>
      <width>59</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>74</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sigma</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>40</y>
      <width>134</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mutation probability</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="populationSize">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>6</y>
      <width>59</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>40</y>
      <width>102</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Max generation</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mutationProbability">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>40</y>
      <width>59</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1072</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>function1</tabstop>
  <tabstop>function2</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>populationSize</tabstop>
  <tabstop>maxGenertation</tabstop>
  <tabstop>crossingProbability</tabstop>
  <tabstop>mutationProbability</tabstop>
  <tabstop>sigma</tabstop>
  <tabstop>tdom</tabstop>
  <tabstop>startButton</tabstop>
  <tabstop>resultsTable</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
